changes/ce/feat-14059.en.md

```diff
@@ -0,0 +1 @@
+Added a new configuration to retainer that allows overriding message expiry intervals defined for each retained message.  This allows garbage collection to remove messages earlier in case of depleted storage.
```

Comment:
```suggestion
Added a new configuration to retainer that allows capping message expiry intervals defined for retained messages.  This allows garbage collection to remove messages earlier in case of depleted storage.
```